doc/gettingstarted.md

```diff
@@ -25,16 +25,46 @@ In this guide we will get everything up and running on a server using Docker.
 
 For now, the web interface Console is not available; in this tutorial, we use the command-line interface (CLI) to manage the network.
 
-Get the latest packages or binaries for your operating system from the [releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on GitHub.
+Keep in mind that in this tutorial the stack is meant to be run from Docker, and you should run only the CLI commands from the host machine.
```

Comment:
Does "keep in mind" refer to an earlier statement in this doc, and if so, which one?

Also, isn't the point of these CLI commands to _not_ run on the host, like stated here, but to manage from your local machine?